velodyne_description8/urdf/VLP-16.urdf.xacro

```diff
@@ -11,11 +11,9 @@
 
     <link name="${name}_base_link">
       <inertial>
-        <mass value="0.83"/>
-        <origin xyz="0 0 0.03585"/>
-        <inertia ixx="${(0.83 * (3.0*0.0516*0.0516 + 0.0717*0.0717)) / 12.0}" ixy="0" ixz="0"
-          iyy="${(0.83 * (3.0*0.0516*0.0516 + 0.0717*0.0717)) / 12.0}" iyz="0"
-          izz="${0.5 * 0.83 * (0.0516*0.0516)}"/>
+        <mass value="0.1"/>
+        <origin xyz="0 0 0"/>
```

Comment:
I'm also wary to change the origin, as this will impact on anyone using this, won't it?